shared/src/message.rs

```diff
@@ -93,3 +93,21 @@ impl<'de> Deserialize<'de> for Message {
         })
     }
 }
+
+#[cfg(feature = "arb")]
+impl quickcheck::Arbitrary for Message {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        Self {
+            to: Address::new_id(u64::arbitrary(g)),
+            from: Address::new_id(u64::arbitrary(g)),
+            version: i64::arbitrary(g),
+            sequence: u64::arbitrary(g),
+            value: TokenAmount::from_atto(u64::arbitrary(g)),
```

Comment:
Let's implement `Arbitrary` for `TokenAmount`.

```suggestion
            value: TokenAmount::arbitrary(g),
```

Reply:
:+1: